easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-2022.07.1+554-foss-2022a-Java-11-R-4.2.1.eb

```
@@ -0,0 +1,111 @@
+easyblock = 'CMakeNinja'
+
+name = 'RStudio-Server'
+version = '2022.07.1+554'
+versionsuffix = '-Java-%(javaver)s-R-%(rver)s'
+local_git_rev = '7872775ebddc40635780ca1ed238934c3345c5de'
+
+homepage = 'https://www.rstudio.com/'
+description = """This is the RStudio Server version.
+RStudio is a set of integrated tools designed to help you be more productive with R.
+
+The server can be started with:
+  rserver --server-daemonize=0 --www-port 8787 --rsession-which-r=$(which R)
```

I agree with @kosl that the longer invocation text is better.

```suggestion
  rserver --server-daemonize=0 --www-port=8787 --rsession-which-r=$(which R) 
--server-user=${USER} --secure-cookie-key-file=${MYTMP}/secure-cookie-key 
--server-data-dir=${MYTMP}/sdd --database-config-file=${MYTMP}/db.conf
```

---

Why? IMO that is to much noise with many parameters which are not required. For example the default paths for user, coockie-key-file etc are fine, some already in a /tmp and maybe even more useful than in a newly created dir which could cause issues when accidentally starting multiple servers. I'd expect users to know better how to configure the server and if not there is documentation for this.

---

User that would like to start rserver on a compute node needs all above parameters. The package is not meant for sysadmin only.